.github/workflows/build.yml

```diff
@@ -147,14 +147,46 @@ jobs:
     permissions:
       contents: write
     steps:
+      - name: Clone repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          fetch-depth: 0
+      - name: Get next version
+        id: get_next_version
+        uses: thenativeweb/get-next-version@f9202b1ab94b345e7ca86da624762a5e17d4cd3e # v2.6.2
+        with:
+          prefix: v   # optional, defaults to ''
+      - name: Show the next version
+        run: |
+          echo ${{ steps.get_next_version.outputs.version }}
+      - name: Install tools
+        run: make install-tools
+      - name: Update VERSION file
+        run: |
+          new_version=${{ steps.get_next_version.outputs.version }}
+          sed -i "s/version: .*/version: ${new_version}/" versions.yaml
+      - name: Set Git user to the commit author #This is so multimod prerelease can get author info
+        run: |
+          git config --global user.name "${{ github.event.head_commit.committer.name }}"
+          git config --global user.email "${{ github.event.head_commit.committer.email }}"
+      - name: Stage changes
+        run: git add .
+      - name: Commit changes
+        run: |
+          git commit -m "Automated commit: prepare for multimod (version ${{ steps.get_next_version.outputs.version }})"
+      - name: Multimod prerelease
+        run: make multimod-prerelease
```

Comment:
Should we be running pre-release here?

Reply:
**➜  multimod git:(main) ./multimod prerelease -h**
Updates version numbers and commits to a new branch for release:
- Checks that the working tree is clean.
- Checks that Git tags do not already exist for the new module set version.
- Switches to a new branch called prerelease_<module set name>_<new version>.
- Updates version.go files, if they exist.
- Updates module versions in all go.mod files.
- Attempts to call 'go mod tidy' in the directory of each modified go.mod file.
- Adds and commits changes to Git branch

Usage:
  versions prerelease [flags]

Flags:
  -a, --all-module-sets              Specify this flag to update versions of modules in all sets listed in the versioning file.
  -b, --commit-to-different-branch   Specify this flag to commit to a different branch. (default true)
  -h, --help                         help for prerelease
  -m, --module-set-names strings     Names of module set whose version is being changed. Each name be listed in the module set versioning YAML. To specify multiple module sets, specify set names as comma-separated values.For example: --module-set-names="mod-set-1,mod-set-2"
  -s, --skip-go-mod-tidy             Specify this flag to skip calling 'go mod tidy'. To be used for debugging purposes. Should not be skipped during actual release.


With the -b=false flag, pre release will commit the changes to the branch.